src/main/java/duke/exception/InvalidRequestException.java

```diff
@@ -0,0 +1,19 @@
+package duke.exception;
+
+import duke.exception.InvalidInputException;
```

Comment:
It appears to me that this import was not used. Maybe you could take a look and check if it could be removed?

Reply:
Thank you for pointing it out! I have deleted the unused import statement.